.github/workflows/cla.yml

```diff
@@ -0,0 +1,29 @@
+name: Verify Contributor License Agreement
+
+on: [pull_request]
+
+jobs:
+  cla-validate:
+
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - run: |
+        echo "::set-env name=CLA_SIGNED::$(grep -q ${{ github.actor }} ./tools/.lp-to-git-user && echo CLA signed || echo CLA not signed)"
```

Comment:
Should we include `": ` in the grep string, to make sure we don't match substrings (or Launchpad usernames)?

Reply:
good thought, and agreed.